contracts/test/committee/SpyCommittee.sol

```diff
@@ -72,4 +72,8 @@ contract SpyCommittee is MasterCopyNonUpgradable, CommitteeI{
         spyDislocation = _dislocation;
         spyProposal = _proposal;
     }
+
+    function quorum() external view returns (uint256) {
+        return uint256(1);
```

Comment:
Please see comment above.
This is not needed as it can be evaluated from committee size

Reply:
Updated